// mediasoup/node/lib/fbs/rtp-parameters/rtp-header-extension-uri.js
"use strict";
// automatically generated by the FlatBuffers compiler, do not modify
Object.defineProperty(exports, "__esModule", { value: true });
exports.RtpHeaderExtensionUri = void 0;
/* eslint-disable @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any, @typescript-eslint/no-non-null-assertion */
var RtpHeaderExtensionUri;
(function (RtpHeaderExtensionUri) {
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["Mid"] = 0] = "Mid";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["RtpStreamId"] = 1] = "RtpStreamId";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["RepairRtpStreamId"] = 2] = "RepairRtpStreamId";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["AudioLevel"] = 3] = "AudioLevel";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["VideoOrientation"] = 4] = "VideoOrientation";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["TimeOffset"] = 5] = "TimeOffset";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["TransportWideCcDraft01"] = 6] = "TransportWideCcDraft01";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["AbsSendTime"] = 7] = "AbsSendTime";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["AbsCaptureTime"] = 8] = "AbsCaptureTime";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["PlayoutDelay"] = 9] = "PlayoutDelay";
    RtpHeaderExtensionUri[RtpHeaderExtensionUri["DependencyDescriptor"] = 10] = "DependencyDescriptor";
})(RtpHeaderExtensionUri || (exports.RtpHeaderExtensionUri = RtpHeaderExtensionUri = {}));
